api/docs/v2/storefront/index.yaml

```diff
@@ -4060,32 +4060,34 @@ paths:
 
         ### Update the Customer information
 
-        ```texttext
```

Comment:
Can we replace `text` with `request`, I was testing this out and it works and looks better:

<img width="573" alt="Screenshot 2021-09-29 at 09 50 00" src="https://user-images.githubusercontent.com/1240766/135235548-773f7439-13db-4eb1-8255-0e1823383975.png">